portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,79 @@
+          </a>
+          <a href="https://github.com/glstewart17">
+            <img src="images/github-logo.png" alt="Github Account">
+          </a>
+        </div>
+      </div>
+      <div id="content">
+        <h2>About Me</h2>
+        <p>I grew up in a small town in Lancaster County, Pennsylvania. During my time there, I engaged in many extracurricular activities and volunteer 
+          organizations: National Honor Society, Science Olympiad, Varsity Volleyball, Varsity Golf, Interact Club, Student Council, and Student Government. 
+          I graduated second in my class of 237 in June 2018. While I took a great deal away from high school education, it lacked computer science classes, 
+          like many schools in the area.
+          
+        <p>Coding was something I always wanted to learn about, but never knew where to start. So, when two former classmates came to me about starting a community organization 
+          that empowers students to learn about computer science, I agreed. The organiztion we co-founded was called Convert to Code. As the other co-founders left for college, 
+          I was left to run Convert to Code for approximately the first year. I was a high school senior with no programming experience running an organization that teaches 
+          students about computer science. By talking with the professionals who agreed to teach the worskshops we organize and helping students during our events, I was able 
+          to learn a great deal about computer science and grow more passionate about programming. While Convert to Code only lasted 3 years, in that time we reached over 200 
+          students through our monthly workshops and a hackathon.
+  
+        <p>I originally applied to college as a Mechanical Engineering major, but after finding my passion for computer science, it was an easy decision to make the switch
+          when I began my first semester at Penn State. In addittion to my classwork, I have taken a leadership role in Penn State's ACM club. One major component of the club
+          is DevPSU, which holds a series of lectures to teach students the software development skills that their classes do not, and hosts a startup program which assigns
+          students into teams and provides them with a real world project to work on, sometimes even for compensation. As the DevPSU Admisistrator, I oversee theses two 
+          offerings and look to provide better continuity between them. In my freetime, I enjoy participating in competitive coding trainings, hackathons, and pickup basketball
+          and volleyball.
+  
+        <p>Thanks to Convert to Code, I was able to start my professional career the summer after my senior year of high school. A former instructor and mentor got me a job as 
+          a site reliability engineer intern at the WebstaurantStore, an online restaurant equitment retailer that generates over a billion dollars in annual revenue. The
```

Comment:
fix sp 'equipment', though I'll be honest I went to Google to see if 'equitment' was a word :)

Reply:
I don't know why I wrote all of that in the shell and not something with spellcheck, I'll definitely be smarter about that in the future.